docs/local-development-setup.md

```diff
@@ -39,20 +41,18 @@ It does feel a bit strange to have dependencies committed, but it's a well worth
 
 ### 2. Bootstrap dependencies
 
-
 #### `node` and `yarn`
 
 First make sure you have `node` and `yarn` and that their versions are compatible with this project.
 This is a common source of issues, best not to skip this step:
 
-* `node -v` should return `16.13.1`\
+- `node -v` should return a version compatible with what's in [.nvmrc](../.nvmrc)\
```

Comment:
Rather than hard-coding the version here, and worrying about it getting out of date, we can just point users directly to `.nvmrc`, which should have the version of `node` used by the project.